cmd/glasskube/cmd/bootstrap.go

```diff
@@ -166,6 +180,8 @@ func init() {
 	bootstrapCmd.Flags().BoolVar(&bootstrapCmdOptions.createDefaultRepository, "create-default-repository",
 		bootstrapCmdOptions.createDefaultRepository,
 		"Toggle creation of the default glasskube package repository")
+	bootstrapCmd.PersistentFlags().BoolVar(&bootstrapCmdOptions.dryrun, "dry-run", false,
+		"simulate the installation of package without actually installing it")
```

Comment:
The description is not accurate, please change it to "Simulate the installation of Glasskube without actually installing it"